### PR TITLE
Add jupyter lite & widgets applications ...

### PR DESCRIPTION
**Pull request by goyalyashpal**
* Jupyter Lite: Fix https://github.com/jupyterlite/jupyterlite/issues/1098
* Jupyter Widgets: Companion the voila & respect width of 3 applications

Destination: https://jupyter.org/try

Source: 
* https://github.com/jupyterlite
* https://ipywidgets.rtfd.io/en/latest/
* https://jupyter.org/widgets

~Please squash these into 1 - the second commit doesn't do anything different - it's just i am not proficient in git~

2023-07-28: it was easy ... "open in codespaces" --> `git rebase -i HEAD~2` --> `reword, fixup` --> `git push --force-with-lease`

### CODE REVIEW

**Comment by goyalyashpal** [comment]
ping @jasongrout 😃

**Comment by goyalyashpal** [comment]
ping @blink1073

**Comment by goyalyashpal** [comment]
I have rebased it to master (using github codespaces)

<details><summary>successful command log just for my own records</summary>



```console
$ git switch master
$ git remote --verbose add -m master upstream https://github.com/jupyter/jupyter.github.io
```

```console
$ git branch -u upstream/master master
branch 'master' set up to track 'upstream/master'.
$ git remote set-head upstream --auto
upstream/HEAD set to master
$ git pull --rebase
Updating 16be457..6454730
Fast-forward
$ git log --oneline -10
$ git switch lite-and-widgets 
Switched to branch 'lite-and-widgets'
Your branch is up to date with 'origin/lite-and-widgets'.
$ git rebase master
Successfully rebased and updated refs/heads/lite-and-widgets.
$ git log --oneline -7
$ git push --force-with-lease
```


<details><summary>Failed commands</summary>

This block contains failed commands - hence commented out
.
```
$ # git pull --rebase upstream
You asked to pull from the remote 'upstream', but did not specify
a branch. Because this is not the default configured remote
for your current branch, you must specify a branch on the command line.
$ # git pull --rebase upstream/master
fatal: 'upstream/master' does not appear to be a git repository
fatal: Could not read from remote repository.
$ # git remote set-head upstream --auto
upstream/HEAD set to master
$ # git pull --rebase
Already up to date.
$ # git fetch master/master
fatal: 'master/master' does not appear to be a git repository
fatal: Could not read from remote repository.
```

</details>

</details>

**Comment by blink1073** [comment]
Hi @goyalyashpal, it looks like the png is corrupted:

image

**Comment by goyalyashpal** [comment]
> it looks like the png is corrupted:
> [screenshot of github diff not rendering for png]

umh, github is not rendering pngs for some 1 to 3 months now, not even in normal repo view. 

is there some other way to verify if it's broken or not?

**Comment by goyalyashpal** [comment]
It looks like this now... the tile size doesn't look uniform..

image

File: try.html [hunk 1]
```diff
@@ -39,6 +39,18 @@
       alt: Python logo - Launch Jupyter Notebook demo Binder
       url: https://jupyter.org/try-jupyter/retro/notebooks/?path=notebooks/Intro.ipynb
 
+    - title: JupyterLite on GitHub Pages
```
Comment (goyalyashpal) [inline try.html:42]:
```suggestion
    - title: JupyterLite (GitHub Pages)
```

File: try.html [hunk 2]
```diff
@@ -39,6 +39,18 @@
...
+      description: JupyterLite (Wasm powered Jupyter) deployed as static GitHub Pages
+      src: try/jupyterlite.png
+      alt: JupyterLite logo - Launch JupyteLite demo Binder
+      url: https://jupyterlite.github.io/demo
+
+    - title: Jupyter Widgets
+      description: Interactive HTML widgets for Jupyter notebooks
```
Comment (goyalyashpal) [inline try.html:49]: wait, working on it.

```suggestion
      description: Interactive HTML widgets for Jupyter notebooks
```

**Comment by goyalyashpal** [comment]
Final would look like this:

**Comment by goyalyashpal** [comment]
@blink1073 it's ready from my side :)

**Review by blink1073 — APPROVED** [review]
Thank you!

**Comment by goyalyashpal** [comment]
hey @blink1073 - just wondering - was there an option to rebase & merge shown for this pr? if yes, then is there some particular reason for doing merge (which adds a merge commit to the history). thanks :)

**Comment by blink1073** [comment]
Good catch, thanks, I've updated our settings.

**Pull request merged**